setup.py

```diff
@@ -46,7 +51,7 @@
     description="a liking app for Django",
     name="pinax-likes",
     long_description=LONG_DESCRIPTION,
-    version="2.2.1",
+    version="2.2.2",
```

Comment:
As per @paltman guidance let's bump this to 3.0.0 after major support matrix changes.